src/Qwiq.Core/QueryDefinition.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Globalization;
+
+namespace Microsoft.Qwiq
+{
+    public class QueryDefinition : IQueryDefinition
+    {
+        public QueryDefinition(Guid id, string name, string queryText)
```

Comment:
`.ctor` in Core should not be public (we don't want folks creating types outside of the Mocks, REST, and SOAP clients)

Reply:
Should it be marked as abstract then? Either way will mark as internal and ensure a mock implementation